### PR TITLE
Added "Entity Parent: Create new record" test; updated selectors, expected and testData files

### PR DESCRIPTION
**Pull request by gmirzoyan**
https://trello.com/c/AocU1dNP

### CODE REVIEW

File: test/specs/entityParent.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        const inputData = testData.parent.inputData;
+        const recordArr = $$(selectors.parent.recordLine).slice(0, 4);
+        expect(recordArr.length === inputData.length).equal(true);
+        for(let i = 0; i < recordArr.length; i++) {
+            expect(recordArr[i].getText()).to.equal(inputData[i]);
+        }
+    });
+    it('08/LP-Should verify showing only one row in the result', () => {
+        expect($(selectors.parent.resultLine).getText()).
+        to.equal(expected.parent.resultLine);
```
Comment (Burdishka) [inline test/specs/entityParent.js:63]: I would say that here you test the functionality of accurately displaying the correct number of records, but not how many records you have on the entity's page.

File: test/specs/entityParent.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    });
+});
+
+
+
+
+
```
Comment (Burdishka) [inline test/specs/entityParent.js:70]: delete these empty lines

**Review by Burdishka — APPROVED** [review]

File: test/specs/entityParent.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+describe("ENTITY PARENT", () => {
+    before(() => {
+        browser.maximizeWindow();
+        browser.url('/');
+        userLogin(testData.userLogin.loginName, testData.userLogin.password);
+    });
+    it('01/LP-Should reset all values', () => {
+        $(selectors.navigationBar.userProfile).click();
+        $(selectors.navigationBar.resetAll).click();
+        expect($(selectors.navigationBar.notification).isDisplayed()).false;
```
[the remark this answers is not in the document]
Comment (SamJulia) [reply test/specs/entityParent.js:16]: That's a good point. In normal way we need to check every places where reset value should reset data and verify all data from such places were reset. But for now let it be like it is.

File: test/specs/entityParent.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        expect($(selectors.parent.parentForm).isDisplayed()).to.equal(true);
+    });
+    it('04/LP-Should type the data into input fields ' +
+        'and select the current date and time', () => {
+        $(selectors.parent.inputString).setValue(testData.parent.inputString);
+        $(selectors.parent.inputText).setValue(testData.parent.inputText);
+        $(selectors.parent.inputInt).setValue(testData.parent.inputInt);
+        $(selectors.parent.inputDecimal).setValue(testData.parent.inputDecimal);
+        $(selectors.parent.inputDate).click();
+        expect($(selectors.parent.inputDate).isDisplayed()).to.equal(true);
```
[the remark this answers is not in the document]
Comment (SamJulia) [reply test/specs/entityParent.js:37]: Agree, if you already click it is displayed. As an option, try to verify the date that was selected.

File: test/specs/entityParent.js (hunk 2 above)
Comment (SamJulia) [reply test/specs/entityParent.js:70]: agree

File: test/specs/entityParent.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        $(selectors.parent.inputDatetime).click();
+        expect($(selectors.parent.inputDatetime).isDisplayed()).to.equal(true);
+    });
+    it('05/LP-Should click on the "SAVE" button and ' +
+        'verify redirection to the List of records page', () => {
+        $(selectors.parent.saveBtn).click();
+        expect($(selectors.parent.listLink).
+        getAttribute(expected.parent.listLinkAttribute)).
```
[the remark this answers is not in the document]
Comment (SamJulia) [reply test/specs/entityParent.js:45]: agree

**Review by SamJulia — COMMENTED** [review]
Please, check comments

File: test/specs/entityParent.js (hunk 2 above)
Comment (SergeiKuznetsov) [inline test/specs/entityParent.js:70]:
```suggestion
```

File: test/specs/entityParent.js [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+        $(selectors.navigationBar.userProfile).click();
+        $(selectors.navigationBar.resetAll).click();
+        expect($(selectors.navigationBar.notification).isDisplayed()).false;
+    });
+    it('02/LP-Should click on the "HOME" link > "Parent" entity ' +
+        'and verify redirection to the "Parent" page', () => {
+        $(selectors.parent.homeLink).scrollIntoView();
+        $(selectors.parent.parentField).click();
+        expect($(selectors.parent.textSign).getText()).
+        to.equal(expected.parent.headerText);
```
Comment (SergeiKuznetsov) [inline test/specs/entityParent.js:23]:
```suggestion
        expect($(selectors.parent.textSign).getText())
            .to.equal(expected.parent.headerText);
```

Comment (SergeiKuznetsov) [reply test/specs/entityParent.js:23]: You should move to the new line with the period and indents.

File: test/specs/entityParent.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (gmirzoyan) [reply test/specs/entityParent.js]: Removed the verifications.Thanks

File: test/specs/entityParent.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (gmirzoyan) [reply test/specs/entityParent.js]: Fixed. Thank you

File: test/specs/entityParent.js (hunk 1 above)
Comment (gmirzoyan) [reply test/specs/entityParent.js]: Created the method validating the number of records

File: test/specs/entityParent.js (hunk 2 above)
Comment (gmirzoyan) [reply test/specs/entityParent.js]: Deleted the additional lines

File: test/specs/entityParent.js (hunk 6 above)
Comment (gmirzoyan) [reply test/specs/entityParent.js]: Fixed. Thank you

**Pull request merged**